lms/djangoapps/certificates/api.py

```diff
@@ -63,16 +63,8 @@ def format_certificate_for_user(username, cert):
             "created": cert.created_date,
             "modified": cert.modified_date,
             "is_passing": is_passing_status(cert.status),
-
-            # NOTE: the download URL is not currently being set for webview certificates.
-            # In the future, we can update this to construct a URL to the webview certificate
-            # for courses that have this feature enabled.
             "is_pdf_certificate": bool(cert.download_url),
-            "download_url": (
-                cert.download_url or get_certificate_url(cert.user.id, cert.course_id)
-                if cert.status == CertificateStatuses.downloadable
-                else None
-            ),
```

Comment:
Why is the `cert.status` checking part of the code now removed?

Reply:
It seemed unnecessary. Do you understand the purpose of that check?

Reply:
I think that check is needed for legacy PDF certificates - where an async process (currently in XQueue) generates the certificate and updates the status to indicate that it's now downloadable.  Unfo, we still support PDF certificates.

Reply:
Okay, I'll add it back in.